packages/fix-hardcoded-paths.patch

```diff
@@ -0,0 +1,12 @@
+diff -uNr calcurse-4.6.0/src/utils.c calcurse-4.6.0.mod/src/utils.c
+--- calcurse-4.6.0/src/utils.c	2020-02-04 23:46:34.000000000 +0200
++++ calcurse-4.6.0.mod/src/utils.c	2020-10-01 20:31:32.951590892 +0300
+@@ -775,7 +775,7 @@
+ 		return P_tmpdir;
```

Comment:
Why that file appeared directly under "packages" directory? It should be inside "calcurse".

Reply:
Oh yeah.. my bad..